index.html

```diff
@@ -12,7 +12,11 @@
         <link rel="stylesheet" href="./styles/reset.css" />
         <link rel="stylesheet" href="./styles/global.css" />
 
+<<<<<<< HEAD
```

Comment:
Be careful when you're resolving merge conflicts on the GitHub web interface. These extra characters will cause your CI to fail and sometimes will cause your app to break entirely. Just something to keep in mind as you move forward in the program. :)

These merge lines are on `line 15, 17, and 19`. Also in your reset.css file on `line 35, 36, and 38`.